packages/video_player/video_player_android/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## Updating pigeon-generated files
+
+If you update files in the pigeons/ directory, run the following
+command in this directory (ignore the errors you get about
+dependencies in the examples directory):
+
+```bash
+flutter pub upgrade
+flutter pub run pigeon --dart_null_safety --input pigeons/messages.dart
```

Comment:
This LGTM, but we should probably have a similar note in `video_player_platform_interface`.

Reply:
This was just copied from `video_player`, but I did forget to copy it to `video_player_platform_interface`. Since it's no longer relevant in `video_player`, I've moved that copy to `video_player_platform_interface`, and added a warning about breaking changes to that copy.